src/psyclone/tests/domain/lfric/lfric_loop_test.py

```diff
@@ -261,39 +260,23 @@ def test_lower_to_language_normal_loop():
     assert loop1.start_expr.symbol.name == "loop1_start"
 
 
-def test_lower_to_language_domain_loop():
-    ''' Tests that we can call lower_to_language_level on a domain LFRicLoop.
-    This test takes an invoke with two consecutive domain kernels and then
-    fuses the 'loops' to verify that the kernels are all still in the right
-    order.
+def test_lower_to_language_null_loop():
+    ''' Tests that we can call lower_to_language_level on a NULL LFRicLoop.
+    The NULL loop is replaced by a dummy loop with a single iteration
     '''
 
     _, invoke = get_invoke("25.1_kern_two_domain.f90", TEST_API, idx=0)
-    # Domain loops cannot be fused with the transformation, so manually
-    # move the two kernels into one domain loop. First detach the second
+    # For this test we want to put 2 kernels inside the same null loop
```

Comment:
This comment needs updating because we no longer manually move the two kernels into a single loop. In fact, I'm not quite sure what is going on so please add comments :-)